### DISCUSSION TITLE
Group rule to update different packages only to the same version

### DISCUSSION THREAD

**Discussion by lovetheguitar** [renovatebot/renovate#28790] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab 

### Please tell us more about your question or problem

Originally asked in https://github.com/renovatebot/pre-commit-hooks/issues/1945:

> I have a package rule to update the used renovatebot docker image and pre-commit validation hook simultaneously.
> 
> Is it possible to ~~either speed up this release here or~~ change the rule somehow that only matching versions (ignoring the docker sha) are chosen?
> 
> ```json
> "packageRules": [
>     {
>       "groupName": "renovate bot docker image and pre-commit validation hook",
>       "groupSlug": "renovate-docker-image-and-pre-commit-hook",
>       "matchPackageNames": ["renovate/renovate", "renovatebot/pre-commit-hooks"]
>     }
>   ]
> ```
> 
> Thanks a lot for your great work. :)

Currently I often get e.g. the pre-commit version updated to `rev: 37.334.1` and the renovate docker image to `CI_RENOVATE_IMAGE: renovate/renovate:37.334.3@sha256:4f69674c08d9612761bdd9158b171a020765e3e704b851aac3eab2f31f52c0aa`.

Can I somehow change the configuration so renovate only upgrades to the latest version available for both? In the case above apparently `37.334.1`.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Unfortunately not yet supported, but discussed here: https://github.com/renovatebot/renovate/issues/9739
